css/style.css

```diff
@@ -34,15 +43,19 @@ header {
 .main-page-h1 {
   font-size: 36px;
   font-weight: 700;
+  padding: 0;
+  margin-bottom: 0;
+  height: 250px;
 }
 
 .main-p {
+  padding: 0;
+  margin-top: 0;
   font-size: 20px;
-  padding: 10px;
 }
 
 .media-h2 {
-  color: blue;
+  color: rgb(0, 89, 255);
```

Comment:
- [ ] [Optional] I also encourage you to take special attention to the element colors so that they match those in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

Reply:
color was set to #7F8CFF;